Readme desc

TangoYankee opened the pull request:
## Issue
#2 Readme Description

## Description
Add:
* contributing guidelines
* instructions to report security issues
* descriptions of the project and organizations

Remove:
* Get Help: current instructions would overlap with contributing instructions

theecrit commented:
Thanks. What's the process for review/merging/commits? I can start detailing out the Contributing.md doc with that info, too, if we have one. :)

TangoYankee commented on README.md:
```diff
@@ -1,20 +1,39 @@
+# Overview
+WOAQ is developing a user-friendly database that allows WOEIP to manage the local air quality (AQ) data they've been collecting for years, visualize that data in meaningful ways, and use this to empower local residents to drive change in their community.
+
+## Process
```
The Readme focuses on the intentions of the project and the broad structure of its introduction. Links to product requirements and their development belong in the contributing document.

theecrit replied:
Okeydoke. Do you want to move that content over, or shall I?

TangoYankee commented on README.md:
```diff
@@ -1,20 +1,39 @@
+* Data upload tool (a flask app hosted on Heroku) [repo](https://github.com/Ethan-bradley/WoaqUploadAppCode) and [user interface](https://whispering-refuge-49854.herokuapp.com/)
+* Data Visualization tool: [repo](https://github.com/openoakland/woaq/) and [user interface](https://openoakland.github.io/woaq/).
+
+## About the data
```
Broad strokes of data collection should be kept for a "User Manual" Overview. However, the details should be moved to the contributing document.

TangoYankee commented on README.md:
```diff
@@ -1,20 +1,39 @@
+* We currently use the [WOAQ Trello board](https://trello.com/invite/b/EBnxZHmx/6e43b909891f622463a67da64dbb8101/west-oakland-air-quality) for general project management. See the [Instructions](https://trello.com/c/msbASe3F) in Trello for how to get started on tasks listed there.
+
 
+### Still TBD:
```
```suggestion
### Under construction:
```

TangoYankee commented on README.md:
```diff
@@ -1,20 +1,39 @@
+
 
+### Still TBD:
 * Distinguish between contributing to the code or to the real-world application
```
```suggestion
```

theecrit replied:
I wasn't sure what this line meant so I left it in, in case it was important to you.

TangoYankee commented on README.md:
```diff
@@ -1,20 +1,39 @@
+### Still TBD:
 * Distinguish between contributing to the code or to the real-world application
 * Those wishing to contribute to the code will link to the [contributing.md](https://github.com/openoakland/woeip/blob/master/.gitignore/contributing.md)
 * Those wishing to contribute to the real-world application will link to whereever the team sees best.
```
```suggestion
```

theecrit replied:
Can you say more about what the difference is between "contributing to the code" versus "the real-world application"? I'm not clear on what the difference is in this context. Ahh, maybe you're referring to contributing to the code versus uploading new data to the live database? We should be a little more clear here.

TangoYankee replied:
Real-world application is a really poor description. I think it's better to say "Product Design." The process of contribution to the code base is different from the process of developing the design. While the two processes need to communicate, they also need to be separated from each other to focus on the challenge each process presents i.e) The github contributing doc gets the code running on someone's computer. Another contributing doc would get a designer spun up on the current design vision.

theecrit commented on README.md:
```diff
@@ -1,20 +1,39 @@
-# Welcome to WOEIP/WOAQ 
-Brief overview of what this readme will outline
+# Welcome! 
+This doc is intended to orient you to the WOAQ-WOEIP partnership and get you up and running as a contributor, if you so choose (we hope you choose).
```
Since once of our goals is to make contributing to this project easy and welcoming, I think it's appropriate here. It's how we've been approaching all our documentation across the board. Do you have an alternative you'd prefer?

TangoYankee replied:
I kept the exclamation point in my re-write. Exclamation points are welcoming, right?

theecrit commented on README.md:
```diff
@@ -1,20 +1,39 @@
+* Test.
+* Improve.
+
+## Tech structure
```
Ok with me.

TangoYankee committed:
Project Description

Add a description for the project and the organizations, a contact for security issues, and brief contributing instructions.
Remove get help section. It would overlap heavily with the contributing instructions.

TangoYankee committed:
todo

add note that a full contributing document is required
